pandas/core/groupby/groupby.py

```diff
@@ -578,15 +579,27 @@ def wrapper(*args, **kwargs):
             # a little trickery for aggregation functions that need an axis
             # argument
             kwargs_with_axis = kwargs.copy()
+            kwargs_wo_axis = kwargs.copy()
+
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
 
+            if name == 'hist' and kwargs_wo_axis.pop('equal_bins', False):
```

Comment:
I don't think another copy of kwargs is the cleanest solution here - can you not just `get` from the kwargs dict instead of popping?

Reply:
In that case `matplotltib.pyplot.hist` throws an error that `equal_bins` is not recognized. An alternative solution that I can think of is to pass `equal_bins` as a named argument to `hist_series` in pandas/plotting/_core.py. But then it will be a dummy variable that's not used within the function. Any suggestions?

Reply:
To clarify a bit more, in python3 (but for some reason not python2), any extra argument passed to `kwargs`will be read in this line (as `kwds`):

https://github.com/pandas-dev/pandas/blob/0370740034978d3a63d4b8e5e2c96ff54e7e08ba/pandas/plotting/_core.py#L2501